token/cli/src/main.rs

```diff
@@ -1861,25 +1831,20 @@ async fn command_gc(
     Ok(result)
 }
 
-async fn command_sync_native(
-    native_account_address: Pubkey,
-    bulk_signers: Vec<Arc<dyn Signer>>,
-    config: &Config<'_>,
-) -> CommandResult {
+async fn command_sync_native(config: &Config<'_>, native_account_address: Pubkey) -> CommandResult {
```

Comment:
note the removal of `bulk_signers`. im 95% sure from looking at `processor.rs` that `sync_native` is permissionless, so signing only with the payer is acceptable in all cases, but i havent tested this by hand